public/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  background-color:#FCFCFC;
+  animation-name: graytocolorimg;
+  animation-duration: 2s;
+}
```

Comment:
may we have a line break between each ruleset please.

i.e there should be lines between each block

```css
block {
  thing: 'thing';
}

block2 {
  thing: 'thing'
}
```